rllib/algorithms/dt/tests/test_segmentation_buffer.py

```diff
@@ -0,0 +1,367 @@
+import unittest
+from typing import Union, List
+
+import numpy as np
+
+import ray
+from ray.rllib.algorithms.dt.segmentation_buffer import (
+    SegmentationBuffer,
+    MultiAgentSegmentationBuffer,
+)
+from ray.rllib.policy.sample_batch import (
+    SampleBatch,
+    MultiAgentBatch,
+    concat_samples,
+    DEFAULT_POLICY_ID,
+)
+from ray.rllib.utils.framework import try_import_tf, try_import_torch
+from ray.rllib.utils.typing import PolicyID
+
+tf1, tf, tfv = try_import_tf()
+torch, _ = try_import_torch()
+
+
+def _generate_episode_batch(max_ep_len, eps_id, obs_dim=8, act_dim=3):
+    """Generate a batch containing one episode."""
```

Comment:
**TODO**: You need to also add dones if you make the split_by_epsiodes(use_dones=True)

Reply:
resolved